### PR TITLE
Modify layout (CSS) to look better

### PR DESCRIPTION
**Pull request by ayamkito**
Add CSS
Change some IDs
Change some Classes

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -12,63 +12,71 @@
...
                     </div>
-                </div>
-                <div class="flexWrap-container-reg">
-                    As this portfolio gets better, I will dedicate this space for the stories of the wild.
+                    <div class="mini-container">
+                        <div class="gallery">
+                            <div>
+                                <button onclick="prevImage">
+                                    << </button>
+                                        <!-- <img src="https://globalgrasshopper.com/wp-content/uploads/2010/03/Kruger-National-Park-1000x664.jpg" -->
```
Comment (darshanmodani-sps) [inline portfolio/src/main/webapp/index.html]: This seems like this isn't correctly commented. You may want to comment the next line as well.

But more generally, as a rule of thumb, before committing any code, you should remove any unused commented code and only keep explanatory comments.

**Review by darshanmodani-sps — APPROVED** [review]
Overall, looks good. Just one suggestion about the commented code. Please fix that before merging the changes :)

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (ayamkito) [reply portfolio/src/main/webapp/index.html]: Okay. well noted. I will resolve that before merging. Thank you.

Comment (ayamkito) [reply portfolio/src/main/webapp/index.html]: It was wrongly commented too. I meant to comment the whole img tag but ended commenting part.

**Pull request merged**